### ISSUE TITLE
Possible PDF error

### ISSUE THREAD

**Issue by tymees** [UiL-OTS-labs/ethics#82]
Found a repeating error in the server log.

Sounds like a table cell is trying to be larger than platypus can handle....

Unless someone has complained, it's not a priority. More something to keep in mind.
I'll take a look at it when I have some free time.
```
[25/May/2020 14:46:56] ERROR [django.request:135] Internal Server Error: /proposals/pdf/652/
Traceback (most recent call last):
  File "/hum/web/etcl.hum.uu.nl/data/etcl/env/lib/python3.4/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/hum/web/etcl.hum.uu.nl/data/etcl/env/lib/python3.4/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/hum/web/etcl.hum.uu.nl/data/etcl/env/lib/python3.4/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/hum/web/etcl.hum.uu.nl/data/etcl/env/lib/python3.4/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/hum/web/etcl.hum.uu.nl/data/etcl/env/lib/python3.4/site-packages/braces/views/_access.py", line 98, in dispatch
    request, *args, **kwargs)
  File "/hum/web/etcl.hum.uu.nl/data/etcl/env/lib/python3.4/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/hum/web/etcl.hum.uu.nl/data/etcl/env/lib/python3.4/site-packages/django/views/generic/detail.py", line 117, in get
    return self.render_to_response(context)
  File "/hum/web/etcl.hum.uu.nl/data/etcl/env/lib/python3.4/site-packages/easy_pdf/views.py", line 64, in render_to_response
    return self.get_pdf_response(context, **response_kwargs)
  File "/hum/web/etcl.hum.uu.nl/data/etcl/env/lib/python3.4/site-packages/easy_pdf/views.py", line 60, in get_pdf_response
    **self.get_pdf_kwargs()
  File "/hum/web/etcl.hum.uu.nl/data/etcl/env/lib/python3.4/site-packages/easy_pdf/rendering.py", line 158, in render_to_pdf_response
    pdf = render_to_pdf(template, context, using=using, encoding=encoding, **kwargs)
  File "/hum/web/etcl.hum.uu.nl/data/etcl/env/lib/python3.4/site-packages/easy_pdf/rendering.py", line 138, in render_to_pdf
    return html_to_pdf(content, encoding, **kwargs)
  File "/hum/web/etcl.hum.uu.nl/data/etcl/env/lib/python3.4/site-packages/easy_pdf/rendering.py", line 65, in html_to_pdf
    link_callback=link_callback, **kwargs)
  File "/hum/web/etcl.hum.uu.nl/data/etcl/env/lib/python3.4/site-packages/xhtml2pdf/document.py", line 142, in pisaDocument
    doc.multiBuild(context.story)
  File "/hum/web/etcl.hum.uu.nl/data/etcl/env/lib/python3.4/site-packages/reportlab/platypus/doctemplate.py", line 1144, in multiBuild
    self.build(tempStory, **buildKwds)
  File "/hum/web/etcl.hum.uu.nl/data/etcl/env/lib/python3.4/site-packages/reportlab/platypus/doctemplate.py", line 1057, in build
    self.handle_flowable(flowables)
  File "/hum/web/etcl.hum.uu.nl/data/etcl/env/lib/python3.4/site-packages/reportlab/platypus/doctemplate.py", line 942, in handle_flowable
    raise LayoutError(ident)
reportlab.platypus.doctemplate.LayoutError: Flowable <PmlTable@0x7F139E7F0FD0 1 rows x 2 cols(tallest row 804)> with cell(0,0) containing
'<PmlKeepInFrame at 0x7f139e7f05c0> size= maxWidth=222.2717x maxHeight=822.0472'(453.54330708661416 x 804.0998505433532), tallest cell 804.1 points,  too large on page 8 in frame 'body'(453.54330708661416 x 714.3307086614175*) of template 'body'
```

**Comment by tymees** [comment]
Ah, just found out this issue was already reported to Jelte

**Comment by tymees** [comment]
Pretty sure the problem is the comments field, which is 20.5k characters and seems to be a copy + paste of a e-mail chain

**Comment by tymees** [comment]
Fixed in 3341a8f547ae23a9f1b05dfc18dd7c83cc16ed42
